freeipa.spec.in

```diff
@@ -291,7 +289,6 @@ Requires: openldap-clients > 2.4.35-4
 Requires: nss >= %{nss_version}
 Requires: nss-tools >= %{nss_version}
 Requires(post): krb5-server >= %{krb5_version}
```

Comment:
Hmm, I guess this needs to be updated for the krb5-kdb-version / krb5-kdb-devel-version stuff.